tools/src/test/java/org/apache/kafka/tools/FeatureCommandTest.java

```diff
@@ -160,28 +154,26 @@ private String outputWithoutEpoch(String output) {
         int pos = output.indexOf("Epoch: ");
         return (pos > 0) ? output.substring(0, pos) : output;
     }
-}
 
-class FeatureCommandUnitTest {
```

Comment:
ditto, adding comments to make it clear the following tests are unit tests.

Reply:
thanks for the advice! just add it back and do feel it clearer as you said 😄